src/App.tsx

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
-import './App.css';
-import { BrowserRouter, Routes, Route } from 'react-router-dom'; // Importa BrowserRouter, Routes y Route
+
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { Home } from './pages/Home';
 
 import Register from './pages/Register';
 
 import Login from './pages/Login';
-import HomeTest from './pages/HomeTest';
 
 const App: React.FC = () => {
   return (
     <BrowserRouter>
       <Routes>
-        <Route path="/" element={<HomeTest />} />{' '}
+        <Route path="/Home" element={<Home />} />{' '}
```

Comment:
pls change "Home" to "home" and keep the lowercase at the beginning of every route. Use kebab case for multiple words.